src/ui/map.ts

```
@@ -476,8 +476,12 @@ class Map extends Camera {
 
         if (typeof window !== 'undefined') {
```

This might not be the PR to address it, but this line bugs me. If `window` is `undefined` then we should be failing very loudly, not silently ignoring it!

---

It would also allow _resizeObserver to be definitely assigned, so later `this._resizeObserver?.disconnect()` could get rid of the `?`

---

I don't have hard feelings for this `if` in general. From my point of view, it can be removed completely.
In what scenario `window` is not defined?